### PR TITLE
added instructions for #174 and #175

### PR DESCRIPTION
**Pull request by andre91a**
#174 Added informations on how to install the license.key

#175 Added informations on how automatically assign a color based on a property value

### CODE REVIEW

File: content/admin/content/getting-started/start.md [hunk 1]
```diff
@@ -55,6 +55,13 @@ docker volume create lke-data
 docker volume create lke-elasticsearch
 ```
 
+Now you need to install your {{lke}} license:
+
+1. if it doesn't already exist, create a folder with the same name as the early created data volume (`lke-data` in our example).
```
Comment (davidrapin) [inline content/admin/content/getting-started/start.md:60]: early/already => typo?

Comment (andre91a) [reply content/admin/content/getting-started/start.md:60]: yes

Comment (davidrapin) [reply content/admin/content/getting-started/start.md:60]: Ok it make sense with "newly" :)

**Comment by davidrapin** [comment]
@andre91a deleting the branch since it was merged.

**Comment by davidrapin** [comment]
@andre91a you will be able to review the changes here: https://dev.linkurio.us/doc-stable/admin-manual/latest/

**Comment by davidrapin** [comment]
@andre91a the PR failed because of an invalid URL.
`Unexpected link URL: "get.linkurio.us" in file "getting-started/start.md" (internal links must start with "/")`
Please add "http://" in from on the URL to fix and open a new PR.

**Pull request merged**